### ISSUE TITLE
ConditionalConfig.cs has a line marked as temporary but is checked in

### ISSUE THREAD

**Issue by mlorbetske** [dotnet/templating#85]
The comment is in this block of code in `ConditionalConfig.CStyleLineCommentsConditionalSetup`

```
// temporary - one time test
IOperationProvider uncomment = new Replacement("//", string.Empty, uncommentOperationId);
IOperationProvider reduceComment = new Replacement("////", "//", reduceCommentsOperationId);
```

Evaluate whether this should be here & either remove the comment or remove the code.

This comment was introduced with commit 03b6f1c0 on 10/13, but the code it applies to was introduced in commit 1c0d0789 on 9/22

**Comment by seancpeters** [comment]
The comment is outdated, I'll remove it in a future pr.

**Comment by seancpeters** [comment]
fixed by #109
